app/src/main/java/com/strv/movies/ui/login/MoviesLoginScreen.kt

```diff
@@ -0,0 +1,188 @@
+    }
+}
+
+@Composable
+private fun LoginInWithSmileButton(navigateToMovieList: () -> Unit) {
+    Button(
+        onClick = { navigateToMovieList() },
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(bottom = 10.dp)
+    ) {
+        Text(stringResource(id = R.string.login_btn))
+    }
+}
+
+@Composable
+private fun PasswordField() {
+    val passwordState = remember { mutableStateOf(TextFieldValue()) }
+    var passwordVisible by rememberSaveable { mutableStateOf(false) }
+
+    TextField(
+        value = passwordState.value,
+        onValueChange = { passwordState.value = it },
+        modifier = Modifier.padding(bottom = 10.dp),
+        isError = false,
+        placeholder = { Text("Password") },
+        label = {
+            Text(text = stringResource(id = R.string.login_password_text_input_field))
+        },
+
+        visualTransformation = if (passwordVisible) VisualTransformation.None else PasswordVisualTransformation(),
+        keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+        trailingIcon = {
+            val image = if (passwordVisible)
+                Icons.Filled.Visibility
+            else Icons.Filled.VisibilityOff
+
+            // Please provide localized description for accessibility services
+            val description = if (passwordVisible) "Hide password" else "Show password"
+
+            IconButton(onClick = { passwordVisible = !passwordVisible }) {
+                Icon(imageVector = image, description)
+            }
+        }
+    )
+}
+
+@Composable
+private fun EmailField() {
+    val emailState = remember { mutableStateOf(TextFieldValue()) }
+    TextField(
+        value = emailState.value,
+        onValueChange = { emailState.value = it },
+        modifier = Modifier.padding(bottom = 10.dp),
+        isError = false,
+        label = {
+            Text(
+                text = stringResource(id = R.string.login_username_text_input_field)
+            )
+        }
+    )
+}
+
+@Composable
+private fun ContinueWithFacebookButton(navigateToMovieList: () -> Unit) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Button(
+            onClick = { navigateToMovieList() },
+//            onClick = { "https://www.facebook.com/index.php" },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 10.dp)
+        ) {
+            Text(
+                modifier = Modifier.padding(start = 10.dp, end = 10.dp),
+                text = stringResource(id = R.string.continue_with_fb)
+            )
+            Icon(
+                painter = painterResource(R.drawable.ic_facebook),
+                contentDescription = stringResource(id = R.string.continue_with_fb),
+                modifier = Modifier.size(24.dp)
+            )
+        }
+    }
+    SignInGmailButton(navigateToMovieList)
```

Comment:
You should probably move this to `LoginElements`

Reply:
well, here, if I do that, then .. design is breaking ... hmmmm
![Screenshot 2022-05-20 224057](https://user-images.githubusercontent.com/36711021/169607590-31131223-e175-4b07-be68-67b007785e70.jpg)

Reply:
hold on, I know the issue...

Reply:
fixe
![Screenshot 2022-05-20 225323](https://user-images.githubusercontent.com/36711021/169609121-67211e1c-764e-4e16-907e-3796e97117dc.jpg)
d